modules/mono/mono_gd/gd_mono_assembly.cpp

```diff
@@ -148,7 +173,7 @@ Error GDMonoAssembly::load(MonoDomain *p_domain) {
 
 	ERR_FAIL_COND_V(loaded, ERR_FILE_ALREADY_IN_USE);
 
-	uint64_t last_modified_time = FileAccess::get_modified_time(path);
+	uint64_t last_modified_time = path.begins_with("res://") ? 0 : FileAccess::get_modified_time(path);
```

Comment:
This would break reloading of assemblies from the editor, since we already load them from `res://.mono/assemblies` and `res://.mono/temp/bin/<Configuration>` and rely on the modified date. Also, AFAIK, keeping it does not do any damage when loading from memory.

Reply:
PCK/ZIP exported project was spamming error messages like:
```
ERROR: _get_modified_time: Method/Function Failed, returning: 0
   At: drivers/unix/file_access_unix.cpp:280.
``` 
I have added PackedData:: check into get_modified_time instead, this shouldn't affect editor.